### ISSUE TITLE
[Feature Request] Plugins and (maybe) A Plugin Marketplace

### ISSUE THREAD

**Issue by TehcJS** [MultiMC/MultiMC5#3283]
System Information
-----------------------------
_N/A_
MultiMC version:
_N/A_

Operating System:
_N/A_
Summary of the issue or suggestion:
----------------------------------------------
Plugins can allow you to add more support for more mod loaders and add more features that won't be added to MultiMC. This includes denied suggestions. This can allow the community to enhance their MultiMC experience without the need of getting their suggestion approved.

What should happen:
------------------------------
_N/A_

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
_N/A_

Suspected cause:
---------------------------
_N/A_

Logs/Screenshots:
----------------------------
_N/A_

Additional Info:
---------------------------
_N/A_

**Comment by Janrupf** [comment]
Probably not going to happen, if you want to extend MultiMC, feel free to contribute. Plugins require a stable API, which MultiMC does not have (and doesn't plan to have in the near future).

**Comment by peterix** [comment]
No.

**Comment by peterix** [comment]
For a slightly longer answer...

1. Suggestions are suggestions. They can lay around for years and then be technically implemented by an unrelated change. I keep them around.
2. Community? People do stick around, but most of them never touch code.
3. Maintaining APIs for people who don't touch the code anyway seems like a waste of time. Not to mention the extra work needed for any future change that dirupts these kinds of things.

In other words... no. Not a chance.
